modules/Movecraft/src/main/java/net/countercraft/movecraft/warfare/assault/AssaultUtils.java

```diff
@@ -12,40 +12,54 @@
 public class AssaultUtils {
 
     public static boolean areDefendersOnline(ProtectedRegion tRegion) {
-        HashSet<UUID> players = new HashSet<>();
-        players.addAll(tRegion.getOwners().getUniqueIds());
         int numOnline = 0;
 
         if(Settings.AssaultRequiredOwnersOnline > 0) {
-            for (UUID playerName : players) {
-                if (Bukkit.getPlayer(playerName) != null) {
+            HashSet<UUID> owners = new HashSet<>();
+            owners.addAll(tRegion.getOwners().getUniqueIds());
```

Comment:
This is unnecessary, just use `tRegion.getOwners().getUniqueIds()` for iteration